extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -71,6 +71,7 @@ public function getContext( Title $title, $skinName ) {
 				'forceProviders' => $this->filterOutEmptyItems( [
 					'directGpt' => $wg->AdDriverForceDirectGptAd,
 					'liftium' => $wg->AdDriverForceLiftiumAd,
+					'turtle' => !!$wg->AdDriverForceTurtleAd,
```

Comment:
Ha! Good catch. No reason for that. We can either cast them or not cast them. There's no point to do both. Sorry about it. The reason is I moved the line from the block where we cast vars to bools to the block where we don't.